packages/shared/locales/en.json

```diff
@@ -3,8 +3,8 @@
         "onboarding": {
             "appSetup": {
                 "welcome": {
-                    "title": "Send, Receive & Manage your {protocol} tokens",
-                    "body": "Firefly is the official wallet software for {protocol}."
+                    "title": "Send, Receive & Manage your {network} tokens",
```

Comment:
I think these changes will require re-translations unless they are done in every locale file